hdmap_from: "NGII"
data_path: "/home/keti-hsh/adc_project/modules/navigation/src/map/hd_map_data/Pangyo HD Map"
link_filename: "A2_LINK"
node_filename: "A1_NODE"
lane_filename: "B2_SURFACELINEMARK"
speedbump_filename: "C4_SPEEDBUMP"
safetysign_filename: "B1_SAFETYSIGN"
surfacemark_filename: "B3_SURFACEMARK"
parkinglot_filename: "A5_PARKINGLOT"
drivewaysection_filename: "A3_DRIVEWAYSECTION"
trafficlight_filename: "C1_TRAFFICLIGHT"
postpoint_filename: "C6_POSTPOINT"
vehicleprotectionsafety_filename: "C3_VEHICLEPROTECTIONSAFETY"
heightbarrier_filename: ""
intersection_filename: "B4_INTERSECTION"
bridge_name: "keti"

camera_param:
  - ["camera0/image_jpeg/compressed", "camera0"]
  - ["camera1/image_jpeg/compressed", "camera1"]

lidar_param:
  - ["/lidar3D_0", "lidar0"]
  # - ["/lidar3D_1", "lidar1"]

radar_param:
  - ["/radar", "radar"]
  # - ["/radar1", "radar1"]